src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Services/SyntaxFacts/CSharpSyntaxKinds.cs

```diff
@@ -31,10 +35,17 @@ protected CSharpSyntaxKinds()
         public int IfDirectiveTrivia => (int)SyntaxKind.IfDirectiveTrivia;
 
         public int CloseBraceToken => (int)SyntaxKind.CloseBraceToken;
+        public int? CloseBracketToken => (int)SyntaxKind.CloseBracketToken;
```

Comment:
why are the bracket tokens nullable?

Reply:
VB does not use `[` in its language.  So it doesn't define tehse values.  So this property needs to be nullable to represent that.  If VB ever adds `[` in its language, then we'd make this non-nullable.